worlds/dark_souls_3/__init__.py

```diff
@@ -489,10 +491,11 @@ def _fill_local_item(
         if not candidate_locations:
             warning(f"Couldn't place \"{name}\" in a valid location for {self.player_name}. Adding it to starting inventory instead.")
             location = next(
-                (location for location in self.multiworld.get_locations(self.player) if location.data.default_item_name == item.name),
+                (location for location in cast(List[DarkSouls3Location], self.multiworld.get_locations(self.player)) if location.data.default_item_name == item.name),
                 None
             )
-            if location: self._replace_with_filler(location)
+            if location:
+                self._replace_with_filler(location)
```

Comment:
I would prefer not to make this change unless there's a style guide prohibition for single-line `if`s

Reply:
This was the only time you had it where the result of the `if` wasn't a `return` or `continue`, but I can just revert it